Cython/Compiler/ModuleNode.py

```diff
@@ -1692,9 +1692,9 @@ def generate_usr_dealloc_call(self, scope, code):
         code.putln("{")
         code.putln("PyObject *etype, *eval, *etb;")
         code.putln("PyErr_Fetch(&etype, &eval, &etb);")
-        code.putln("++Py_REFCNT(o);")
+        code.putln("Py_INCREF(o);")
         code.putln("%s(o);" % entry.func_cname)
-        code.putln("--Py_REFCNT(o);")
+        code.putln("Py_DECREF(o);")
```

Comment:
The thing is, this touches the deallocation machinery, and we're in deallocation code already here. The code pretty intentionally only changes the refcount (to temporarily keep the object alive / resurrect it) for calling into (arbitrary) user defined code, before continuing to clean it up.

Reply:
Oh I see, I modified my PR to add __Pyx_SET_REFCNT() function instead.